tests/unit/wrapper/test_statsmodels_wrapper.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import xarray as xr
+from statsmodels.tsa.ar_model import AutoReg
+
+from pywatts.wrapper.statsmodels_wrapper import StatsmodelsWrapper
+
+
+class TestStatsmodelsWrapper(unittest.TestCase):
```

Comment:
Please add tests for loading and saving the wrapper.

Reply:
First draft done